bot/resources/tags/return.md

```diff
@@ -1,35 +1,32 @@
 **Return Statement**
 
-When calling a function, you'll often want it to give you a value back. In order to do that, you must `return` it. The reason for this is because functions have their own scope. Any values defined within the function body are inaccessible outside of that function.
-
-*For more information about scope, see `!tags scope`*
+A value created inside a function can't be used outside of it unless you `return` it.
```

Comment:
Personally, I think this change makes it harder to understand what exactly a return statement is. My opinion is to revert this change

Reply:
The target audience for this tag are those who are so unfamiliar with programming that they do not know what a return statement is or does, and who may not be proficient in English. The expected use case is to resolve confusion when beginners print the result of the function's computation instead of returning it.

While there might be beginners who expect variables defined in the function to be written to the global scope (especially if there is already a variable with the same name), I think that needs to be a separate discussion, and there is indeed a separate scope tag. But I do not think dedicating an extra line to "Run this other command to get even more information" will inspire anyone to do that, and would instead cost vertical space towards eyes glazing over.

Reply:
This sentence alone might not explain how `return` works, but it does point a beginner in the right direction. If they want something from inside a function, this motivates them to learn how `return` works.

Reply:
I agree with @camcaswell, the point is to simplify usage for early beginners as much as possible.
And beginners usually have a sense of "hesitation" and/or "fear" when reading what something does, and we want to help build confidence & understanding in the simplest way.
This should help them get started just fine.